client/components/tooltip/tooltip-base.tsx

```diff
@@ -32,7 +55,7 @@ const useHideDelay = (
 
 	// hide delay
 	useEffect( () => {
-		let timer = null;
+		let timer: ReturnType< typeof setTimeout > | null = null;
```

Comment:
Used ReturnType here, since this type is different across Node/Browser environments.